services/movie_session.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall_id=cinema_hall_id,
+                                movie_id=movie_id)
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession:
```

Comment:
The function `get_movies_sessions` is expected to return a `QuerySet` of `MovieSession` objects, not a single `MovieSession` object. Please, correct the return type in the function signature.